### DISCUSSION TITLE
Can I use npx nuxi preview in production w/ pm2 process manager

### DISCUSSION THREAD

**Discussion by starter-dev** [nuxt/nuxt#19864] [Questions]
What I've done is I use `pm2` process manager to start my Nuxt application using `npm run preview`. It will then start previewing my Nuxt Application in default port `3000`.

``` javascript
module.exports = {
  apps : [{
      name: 'nuxt-application-name',
      script: "start",
      args: "npm run preview",
   }]
}
```

I guarantee that it works! But I just want to know if this production approach is right. Thank you!

**Comment by danielroe** [comment]
No, you should not use `nuxi preview` in production. Instead use `node .output/server/index.mjs`

**Reply by starter-dev** [reply]
Thank you for the reply. But the in deployment, I didn't run `npm run build`, instead I run `npx nuxi generate`
